src/ed25519.rs

```diff
@@ -449,6 +431,26 @@ impl Keypair {
     }
 }
 
+impl Signer<Signature> for Keypair {
```

Comment:
This is the main interop trait for signing. It's generic around the return value (`ed25519::Signature`, in this case)